syntax = "proto3";

package todoPackage;

service Todo {
  rpc CreateTodo(TodoItem) returns (TodoItem);
  rpc ReadTodos(void) returns (TodoItems);
}

message void {}
message TodoItem {
  int32 id = 1;
  string text = 2;
}

message TodoItems {
  repeated TodoItem items = 1;
}
